### PR TITLE
Add property

### PR DESCRIPTION
**Pull request by KirthiNair**

### CODE REVIEW

File: src/browser/modules/EditorInfo/DisplayNodeDetails.jsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import React, { useState, useEffect } from 'react'
 import PropTypes from 'prop-types'
+import { withBus } from 'react-suber'
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/DisplayNodeDetails.jsx]: unused import?

File: src/browser/modules/EditorInfo/DisplayNodeDetails.jsx [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+import { connect } from 'react-redux'
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/DisplayNodeDetails.jsx]: unused import?

File: src/browser/modules/EditorInfo/DisplayNodeDetails.jsx [hunk 3]
```diff
@@ -14,7 +16,8 @@ import { BinIcon } from 'browser-components/icons/Icons'
 import { ConfirmationButton } from 'browser-components/buttons/ConfirmationButton'
 import { DisplayProperties } from '../EditorInfo/DisplayProperties'
 import { ExpandRelationshipDetails } from './ExpandRelationshipDetails'
-
+// import { PlusIcon } from "browser-components/icons/Icons";
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/DisplayNodeDetails.jsx]: remove commented code

File: src/shared/modules/itemEditor/itemEditorDuck.js [hunk 4]
```diff
@@ -103,6 +103,11 @@ export const handleEditEntityEpic = (action$, store) =>
         }) RETURN a, ((a)-->()) , ((a)<--())`
         break
       case 'update':
+        cmd = `MATCH (a:${action.editPayload.label} {${
```
Comment (rahul-tarafdar-synerzip) [inline src/shared/modules/itemEditor/itemEditorDuck.js]: use node id for match

File: src/browser/modules/EditorInfo/AddProperty.jsx [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+          props.editEntityAction(
+            {
+              label: props.labels,
+              existingKey: Object.keys(props.properties)[0],
+              existingValue: Object.values(props.properties)[0],
+              key: myState.newProperties.key,
+              value: myState.newProperties.propValue
+            },
+            'update',
+            'node'
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/AddProperty.jsx]: use nodeProperties

File: src/browser/modules/EditorInfo/AddProperty.jsx [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+          handleToggle(!textField)
+          props.editEntityAction(
+            {
+              label: props.labels,
+              existingKey: Object.keys(props.properties)[0],
+              existingValue: Object.values(props.properties)[0],
+              key: myState.newProperties.key,
+              value: myState.newProperties.propValue
+            },
+            'update',
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/AddProperty.jsx]: user create

**Review by rahul-tarafdar-synerzip — CHANGES_REQUESTED** [review]

File: src/browser/modules/EditorInfo/DisplayNodeDetails.jsx (hunk 1 above)
Comment (KirthiNair) [reply src/browser/modules/EditorInfo/DisplayNodeDetails.jsx]: done

File: src/browser/modules/EditorInfo/DisplayNodeDetails.jsx (hunk 2 above)
Comment (KirthiNair) [reply src/browser/modules/EditorInfo/DisplayNodeDetails.jsx]: done

File: src/browser/modules/EditorInfo/DisplayNodeDetails.jsx (hunk 3 above)
Comment (KirthiNair) [reply src/browser/modules/EditorInfo/DisplayNodeDetails.jsx]: done

File: src/shared/modules/itemEditor/itemEditorDuck.js (hunk 4 above)
Comment (KirthiNair) [reply src/shared/modules/itemEditor/itemEditorDuck.js]: done

File: src/browser/modules/EditorInfo/AddProperty.jsx (hunk 5 above)
Comment (KirthiNair) [reply src/browser/modules/EditorInfo/AddProperty.jsx]: done

File: src/browser/modules/EditorInfo/AddProperty.jsx (hunk 6 above)
Comment (KirthiNair) [reply src/browser/modules/EditorInfo/AddProperty.jsx]: done

File: src/browser/modules/EditorInfo/AddProperty.jsx [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+    let newState = _.cloneDeep(myState)
+    updatePropertiesState({
+      ...newState,
+      newProperties: {
+        ...newState.newProperties,
+        [key1]: getStringValue(value)
+      }
+    })
+  }
+  function DropDownContents (props) {
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/AddProperty.jsx]: get this out of AddProperty component.

**Review by rahul-tarafdar-synerzip — CHANGES_REQUESTED** [review]

File: src/browser/modules/EditorInfo/AddProperty.jsx (hunk 7 above)
Comment (KirthiNair) [reply src/browser/modules/EditorInfo/AddProperty.jsx]: done

**Review by rahul-tarafdar-synerzip — APPROVED** [review]

**Pull request merged**